layouts/partials/providers.html

```diff
@@ -0,0 +1,15 @@
+{{ $here    := $.Page.RelPermalink }}
+{{ $providers := where site.RegularPages ".CurrentSection.Title" "Authentication Providers" }}
+{{ $version := index (split $.Page.File.Path "/") 1 }}
+<div class="buttons has-extra-top-margin">
+  {{ range $providers }}
+  {{ $isHere      := eq .RelPermalink $here }}
```

Comment:
`$isHere` isn't relevant for providers AFAICT.

Reply:
Done!